File: Core.API/ViewModels/SecurityVM.cs
using Core.Extensions;
using Core.Models;
using System.Collections.Generic;
using System.Linq;

namespace Core.ViewModels
{
    public class SecurityVM : FeaturePolicy
    {
        public bool AllPermission { get; set; }
        public string[] RecordIds { get; set; }
        public string StrRecordIds => RecordIds.Select(x => $"\"{x}\"").Combine();
        public List<FeaturePolicy> FeaturePolicy { get; set; }
    }
}

File: frontend/lib/models/securityVM.js
import { FeaturePolicy } from "./featurePolicy.js";

export class SecurityVM extends FeaturePolicy {
    allPermission = false;
    /** @type {string[]} */
    recordIds = [];
    get strRecordIds() {
        return this.recordIds.map(x => `"${x}"`).join();
    }
    /** @type {FeaturePolicy[]} */
    featurePolicy = [];
}